.gitignore

```diff
@@ -0,0 +1,24 @@
+*.h5
```

Comment:
Let's add some comments!

Reply:
I've added  comments throughout this file.